modules/hcp-create-infraenv.adoc

```diff
@@ -0,0 +1,84 @@
+// Module included in the following assemblies:
+//
+// * hosted_control_planes/hcp-deploy/hcp-deploy-ibm-power.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="hcp-create-infraenv_{context}"]
+= Create Infraenv
```

Comment:
Create needs to be Creating
What is "Infraenv"? This needs one of the following changes:

```suggestion
= Creating infrastructure environment resources
```
(just a random guess what infraenv might be short for)

```suggestion
= Creating `infraenv` custom resources
```